frontend/src/components/import-button.js

```diff
@@ -68,7 +80,7 @@ export function ImportDialog({
         if (onFileChange instanceof Function) {
             onFileChange(fileContents);
         }
-    }, [fileContents, onFileChange]);
+    }, [fileContents, onFileChange, formElement]);
```

Comment:
What? You cannot track regular local variables. Only things set by react hooks can be listed here...

Reply:
This still seems to be an issue...